admin_pages/messages/Messages_Template_List_Table.class.php

```diff
@@ -348,4 +349,18 @@ protected function _get_context_links(EE_Message_Template_Group $item)
 
         return sprintf('<strong>%s:</strong> ', ucwords($c_label['plural'])) . implode(' | ', $ctxt);
     }
+
+
+    /**
+     * Returns the actions for the messenger column.
+     *
+     * Note: Children classes may override this so do not remove it.
+     *
+     * @param EE_Message_Template_Group $item
+     * @return array
+     */
+    protected function _get_actions_for_messenger_column(EE_Message_Template_Group $item)
+    {
+        return [];
```

Comment:
The `Custom_Message_Template_List_Table` class over-rides this method and adds its trash/restore links.  It's left in this class because the messenger column is inherited from it.